pytorch_lightning/loggers/neptune.py

```diff
@@ -114,160 +113,214 @@ def any_lightning_module_function_or_hook(self):
-        offline_mode: Optional default ``False``. If ``True`` no logs will be sent
-            to Neptune. Usually used for debug purposes.
-        close_after_fit: Optional default ``True``. If ``False`` the experiment
-            will not be closed after training and additional metrics,
-            images or artifacts can be logged. Also, remember to close the experiment explicitly
-            by running ``neptune_logger.experiment.stop()``.
-        experiment_name: Optional. Editable name of the experiment.
-            Name is displayed in the experiment’s Details (Metadata section) and
-            in experiments view as a column.
-        experiment_id: Optional. Default is ``None``. The ID of the existing experiment.
-            If specified, connect to experiment with experiment_id in project_name.
-            Input arguments "experiment_name", "params", "properties" and "tags" will be overriden based
-            on fetched experiment data.
+        close_after_fit: Optional default ``True``.
+            If ``False`` the run will not be closed after training
+            and additional metrics, images or artifacts can be logged.
+        name: Optional. Editable name of the run.
+            Run name appears in the "all metadata/sys" section in Neptune UI.
+        run: Optional. Default is ``None``. The ID of the existing run.
+            If specified (e.g. 'ABC-42'), connect to run with `sys/id` in project_name.
+            Input argument "name" will be overridden based on fetched run data.
         prefix: A string to put at the beginning of metric keys.
-        \**kwargs: Additional arguments like `params`, `tags`, `properties`, etc. used by
-            :func:`neptune.Session.create_experiment` can be passed as keyword arguments in this logger.
+        \**kwargs: Additional arguments like ``tags``, ``description``, ``capture_stdout``, ``capture_stderr`` etc.
+            used when run is created.
 
     Raises:
         ImportError:
-            If required Neptune package is not installed on the device.
+            If required Neptune package in version >=0.9 is not installed on the device.
+        TypeError:
+            If configured project has not been migrated to new structure yet.
     """
 
-    LOGGER_JOIN_CHAR = '-'
+    LOGGER_JOIN_CHAR = '/'
 
     def __init__(
-        self,
-        api_key: Optional[str] = None,
-        project_name: Optional[str] = None,
-        close_after_fit: Optional[bool] = True,
-        offline_mode: bool = False,
-        experiment_name: Optional[str] = None,
-        experiment_id: Optional[str] = None,
-        prefix: str = '',
-        **kwargs
-    ):
+            self,
+            api_key: Optional[str] = None,
+            project: Optional[str] = None,
+            close_after_fit: Optional[bool] = True,
+            name: Optional[str] = None,
+            run: Optional[str] = None,
+            prefix: str = '',
+            **neptune_run_kwargs):
```

Comment:
ok this will be compatibility/breaking change as users would import the sea class as before but no it has very different args